octoprint_octorelay/__init__.py

```diff
@@ -196,8 +196,11 @@ def toggle_relay(self, index, target: Optional[bool] = None):
             f"Toggling relay {index} on pin {pin}" if target is None else
             f"Turning the relay {index} {'ON' if target else 'OFF'} (pin {pin})"
         )
-        cmd = settings["cmd_on" if relay.toggle(target) else "cmd_off"]
+        state = relay.toggle(target)
+        cmd = settings["cmd_on" if state else "cmd_off"]
         self.run_system_command(cmd)
+        if state:
+            self.handle_plugin_event(TURNED_ON)
```

Comment:
- [ ] here is the problem: THIS kind of event is only related to the particular relay, it has a scope, or payload, which should be handled by `handle_plugin_event`